tests/run.sh

```diff
@@ -28,6 +28,7 @@ integration-test)
 browser-test)
     shift
     wait_for_server
+    wget -q --tries=180 --retry-connrefused --waitretry=1 -O /dev/null http://selenium:4444/wd/hub/status || (echo "Can't reach Selenium" && exit 1)
```

Comment:
I don't know if this actually solves the root issue (waiting for the selenium container to be ready), but the CI suite just three times in a row without a failure.